get_lc_instance_type.py

```diff
@@ -0,0 +1,34 @@
+from utils import http_error_handling
+import boto3
+from fleece.xray import monkey_patch_botocore_for_xray
+import os
+
+monkey_patch_botocore_for_xray()
+
+cache = dict()
+
+
+def get_lc_instance_type(client, lc_name):
+    desc = client.describe_launch_configurations(
+        LaunchConfigurationNames=[lc_name]
+    )
+    if not len(desc['LaunchConfigurations']):
```

Comment:
Could `LaunchConfigurations` be missing in `desc`? If so it'll raise a `KeyError`

Reply:
always returns an array, even if empty:
```
saws> aws autoscaling describe-launch-configurations --region us-east-2
{
    "LaunchConfigurations": []
}

```